### ISSUE TITLE
does not detect git-annex version correctly?

### ISSUE THREAD

**Issue by yarikoptic** [G-Node/gin-cli#121]
Greetings from the datalad folks!  Great to see lots of seems cool stuff being done in the gin land (last time we chatted with Thomas was a few years back I think).  So decided to checkout the datasets, but fell into some kind of an issue with git annex version detection:

```shell
$> gin get steffi/Kleineidam_et_al_2017    
ERROR The GIN Client requires git-annex 6.20160126 or newer

$> git annex version
git-annex version: 6.20171114+gitg5e6c3ba30-1~ndall+1
```
what could be the catch? (this version of git-annex comes from neurodebian's git-annex-standalone package)

**Comment by cgars** [comment]
Hey Yarik!

Great to hear from you guys. Its probably even longer since we talked (>2 years).

To the topic:
The problem seems to be that the library we use to do the version checking really does not like that version string... (which the client should report and not just die print wrong error message).

We will fix it asap!

Thanks for the awesome annex package btw. and your bug reporting!

**Comment by achilleas-k** [comment]
Thanks for the report. Setting this to top priority.

**Comment by achilleas-k** [comment]
Pull request #124 should fix this and it will be included in release v0.12.

**Comment by yarikoptic** [comment]
great, thank you!
